strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -187,6 +194,21 @@ public Validator localValidatorFactoryBean()
     {
         return new LocalValidatorFactoryBean();
     }
+    
+    @Bean("loggingMgmtDirectoryListingService")
```

Comment:
@saratbeesa can you please use `@Qualifier` annotation?

Reply:
I think both the notations have the same effect, why do you prefer using `Qualifier` at the bean definition ?

Reply:
It's a matter of convention we've adopted. It's better to be consistent. Thanks! :)

Reply:
@saratbeesa please have a look at [`@OrientDb`](https://github.com/strongbox/strongbox/blob/master/strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/OrientDbUserService.java#L201) as example

Reply:
@saratbeesa can you please follow above example?